test/retry-with-a-custom-handler.test.js

```diff
@@ -6,10 +6,11 @@ const From = require('..')
 const http = require('node:http')
 const got = require('got')
 
-function serverWithCustomError (stopAfter, statusCodeToFailOn) {
+function serverWithCustomError (stopAfter, statusCodeToFailOn, closesSocket) {
```

Comment:
I would rename it to `closeSocket`

Reply:
I've made the change but curious if you could expand on why.

Reply:
Because closeSocket is imperative and tells the function to do something, while closesSocket is passive indicative or sounds like a question.